app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const test = require('./routes/test');
+
+const app = express();
+
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json());
+
+app.get('/', (req, res) => res.json({msg: 'Wannabook!'}));
+
+app.use('/test', test);
+const port = process.env.PORT || 5000;
```

Comment:
We should later outsource port names and other stuff into configs.
See https://wannabook.atlassian.net/browse/WNB-70

Reply:
Please, use [config](https://www.npmjs.com/package/config) package to handle different enviroments.

Reply:
Yes, agree. Could you create some very basic config in this repo for now?